app/src/main/AndroidManifest.xml

```diff
@@ -50,22 +50,29 @@
         <activity
             android:name=".skills.SkillsActivity"
             android:configChanges="orientation|screenSize"
-            android:theme="@style/PreferencesThemeLight">
+            android:theme="@style/PreferencesThemeLight"
+            >
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="org.fossasia.susi.ai.chat.ChatActivity" />
         </activity>
 
         <activity
-            android:name=".signup.SignUpActivity"
+             android:name=".signup.SignUpActivity"
             android:theme="@style/SignUp_Theme"
             android:windowSoftInputMode="stateVisible|adjustResize"
-            tools:replace="android:theme" />
+            tools:replace="android:theme"
+            >
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value=".skills.SkillsActivity" />
```

Comment:
Completely wrong solution and we don't need these enhancements

Reply:
Sir,should I remove revert the changes in Manifest file ?

Reply:
Yes